webpack.config.js

```diff
@@ -1,38 +1,94 @@
-const HtmlWebPackPlugin = require("html-webpack-plugin");
```

Comment:
look at another PR

Reply:
Fixed in another PR and merged in this brunch